### PR TITLE
Add support for `fw-info`; Bump LispBM

### PR DESCRIPTION
**Pull request by laxsjo**
This PR adds the new `fw-info`, along with the new VESC command `COMM_FW_INFO` which it uses.

There are also three other PRs for all of the VESC firmware repos.
- https://github.com/vedderb/bldc/pull/791
- https://github.com/vedderb/vesc_bms_fw/pull/22
- https://github.com/vedderb/vesc_gpstm/pull/2

### CODE REVIEW

File: main/lispBM/src/lbm_memory.c [hunk 1]
```diff
@@ -307,7 +307,7 @@ static lbm_uint *lbm_memory_allocate_internal(lbm_uint num_words) {
 }
 
 lbm_uint *lbm_memory_allocate(lbm_uint num_words) {
-  if (memory_num_free - num_words < memory_reserve_level) {
+    if (memory_num_free - num_words < memory_reserve_level) {
```
Comment (vedderb) [inline main/lispBM/src/lbm_memory.c:310]: Is this a mistake? :)

Comment (laxsjo) [reply main/lispBM/src/lbm_memory.c:310]: Oh no, that looks like one! Will fix once I'm back at my computer

Comment (laxsjo) [reply main/lispBM/src/lbm_memory.c]: Force pushed a fix!

**Comment by laxsjo** [comment]
I took the oportunity to bump lispBM while I'm at it, mainly to include the fix of `recv-to`. I hope that isn't a problem.

**Pull request merged**